content/en/docs/installation/upgrading/remove-deprecated-apis.md

```diff
@@ -14,27 +14,110 @@ We have deprecated the following cert-manager APIs:
+You can verify the storage versions on CRD status fields by running `kubectl get crd <crd-name>  -ojsonpath='{.status.storedVersions}`, but we are not aware of any straightforward way how to verify what versions of resources are stored in `etcd`.
+{{% /alert %}}
+
+{{% alert title="Warning" color="warning" %}}
+The following instructions are a loose recap of the [official Kubernetes documentation for updating Custom Resource Definitions](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#writing-reading-and-updating-versioned-customresourcedefinition-objects). We recommend that you use the official Kubernetes documentation as a source of truth.
+{{% /alert %}}
+
+### Upgrading existing cert-manager resources and CustomResourceDefinitions
 
 1. Familiarize yourself with the [official Kubernetes documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#writing-reading-and-updating-versioned-customresourcedefinition-objects) on CRD versioning.
-2. Make sure your installed cert-manager deployment is `v1.0.0` or later.
 
-3. Make sure any version-controlled cert-manager custom resource config files that still use the deprecated APIs are updated to use the `cert-manager.io/v1` API and re-applied. This should update stored versions of the given resources.
+2. Make sure your cert-manager deployment is at version `v1.0` or later.
+
+3. Make sure any version-controlled cert-manager custom resource config files that still use the deprecated APIs are updated to use the `cert-manager.io/v1` API and re-applied. This should update stored versions of the given resources. You can use the [cmctl convert](https://cert-manager.io/docs/usage/cmctl/#convert) tool to convert manifests.
+
+4. Make sure that any non-version controlled cert-manager resources (e.g. `Certificate`s created by ingress-shim, `CertificateRequest`s, etc.) are updated to `v1`. The Kubernetes documentation describes two alternative ways how to do that:
+
+A. Use [storage version migrator](https://github.com/kubernetes-sigs/kube-storage-version-migrator)
+We have tested Storage Version Migrator [`v0.0.5`](https://github.com/kubernetes-sigs/kube-storage-version-migrator/releases/tag/v0.0.5):
+- Follow the [storage version migrator installation instructions](https://github.com/kubernetes-sigs/kube-storage-version-migrator/blob/master/USER_GUIDE.md). (To use the published `v0.0.5` storage migrator images, modify the command that creates local manifests to `make local-manifests REGISTRY=eu.gcr.io/k8s-artifacts-prod/storage-migrator VERSION=v0.0.5`). Alternatively, if using managed Kubernetes, you may already have Storage Migrator installed- look for `storageversionmigrations.migration.k8s.io` and `storagestates.migration.k8s.io` `CustomResourceDefinition`s in cluster to verify if it installed.
```

Comment:
The bit about using v0.0.5 images is needed as the installation instructions for storage migrator doesn't include that part https://github.com/kubernetes-sigs/kube-storage-version-migrator/pull/98